backend/src/main/java/com/zeros/backend/RestaurantFidelityRewarder.java

```diff
@@ -0,0 +1,7 @@
+package com.zeros.backend;
+
+public interface RestaurantFidelityRewarder {
+
+    void incrementRestaurantCounter(Long restaurantId);
+
```

Comment:
i think that we give a lot of implementation details in this name of function :( 
`incrementRestaurantCounter `